public/js/selfoss-base.js

```diff
@@ -20,13 +20,18 @@ var selfoss = {
         source: '',
         ajax: true
     },
-
+    
```

Comment:
Whitespace is used for indentation, what is it for when there is nothing to indent? OK, I’ll leave it at this and make a pull request cleaning the coding style of the whole project later.